integration/examples/jib-sync/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '2.0.5.RELEASE'
+    id 'org.springframework.boot' version '2.6.15'
```

Comment:
I'm curious why you settled on 2.6.15? Why not just upgrade to the latest version for all these plugins, which will hopefully mean it takes longer before we run into this again... you can see the latest versions by searching for these plugins on https://plugins.gradle.org/